ci/environment-3.10.yaml

```diff
@@ -10,6 +10,6 @@ dependencies:
   - pyarrow
   - pytest
   - gcsfs
-  - grpcio<1.45
```

Comment:
If things pass, I wonder if we need it as a explicit dependency, I can't recall exactly but I have the hunch that `grpcio` gets installed with google-cloud-bigquery, in fact I think it is see https://github.com/googleapis/python-bigquery/blob/main/setup.py

We might be able to remove it all the way if we do not need the pin.

Reply:
👍